src/mono/wasm/Wasm.Build.Tests/TestAppScenarios/SignalRClientTests.cs

```diff
@@ -47,14 +47,23 @@ public async Task SignalRPassMessages(string config, string transport)
             ServerEnvironment: new Dictionary<string, string> { ["ASPNETCORE_ENVIRONMENT"] = "Development" },
             BrowserPath: "/chat",
             BrowserQueryString: new Dictionary<string, string> { ["transport"] = transport, ["message"] = "ping" },
-            OnPageLoaded: async page => await page.ClickAsync("button#connectButton"),
-            OnServerMessage: (msg) => { serverOutput.Add(msg); },
+            OnServerMessage: (msg) => serverOutput.Add(msg),
             OnConsoleMessage: async (page, msg) =>
             {
                 consoleOutput.Add(msg.Text);
                 if (msg.Text.Contains("TestOutput ->"))
                     _testOutput.WriteLine(msg.Text);
 
+                // prevent timeouts with [Long Running Test] on error
+                if (msg.Text.Contains("[ERROR]"))
+                    throw new Exception($"{_testOutput}\n{msg.Text}");
```

Comment:
nit: Since `_testOutput` might be multiple lines, adding `msg.Text` before it might produce a more readable output (maybe) 🤷‍♂️

Reply:
Right, it can be corrected in case this operation will be at all helpful. Generally it already should log the debug messages anyway when we timeout but it does not e.g. see [log](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-100074-merge-ce455e8c684c452684/Workloads-NoWebcil-Wasm.Build.Tests.TestAppScenarios.SignalRClientTests/1/console.be50248e.log?helixlogtype=result), even though we set `builder.Logging.SetMinimumLevel(LogLevel.Debug);`, so I don't believe much in the effectiveness of this one